config/src/test/java/org/springframework/security/config/annotation/method/configuration/PrePostMethodSecurityConfigurationTests.java

```diff
@@ -462,7 +462,7 @@ public void methodSecurityAdminWhenRoleHierarchyBeanAvailableThenUses() {
 	@WithMockUser
 	@Test
 	public void methodSecurityUserWhenRoleHierarchyBeanAvailableThenUses() {
-		this.spring.register(RoleHierarchyConfig.class, MethodSecurityServiceConfig.class).autowire();
+		this.spring.register(RoleHierarchyConfig.class, MethodSecurityServiceConfig.class, AuthorizationEventPublisherConfig.class).autowire();
```

Comment:
Please place this into a separate test so that this existing test remains the same as before.

Reply:
Done